apidoc/Titanium/UI/Android/DrawerLayout.yml

```diff
@@ -16,6 +16,27 @@ since: "6.2.0"
 platforms: [android]
 
 properties:
+  - name: GRAVITY_BOTH
+    summary: |
+        Use with [DrawerLayout.drawerLockMode](Titanium.UI.Android.DrawerLayout.drawerGravity) to specify the gravity.
+    type: Number
+    permission: read-only
+    since: "8.1.0"
```

Comment:
These values clearly need to be update for whatever version this ends up in. The JIRA ticket is targeting 8.1.1 - which seems wrong to me since this is an enhancement, not a bug fix.